### PR TITLE
WIP: Relates to #81. Update to Electron v4.1.0

### PR DESCRIPTION
**Pull request by ltfschoen**
* Incorporates review comments from the PR for Electron v3.1.0 https://github.com/paritytech/js-libs/pull/90

* fether terminal tab
  * check that version numbers imported into Fether packages (i.e. `"@parity/electron": "^3.0.1",`)
  match the version number in each js-lib package's package.json version number
  * Note: use 3.0.1 until this issue is resolved https://github.com/paritytech/js-libs/issues/58

```
nvm use 10.11.0;
cd ~/code/src/paritytech/fether;
git checkout luke-309-electron-4;
yarn; yarn build;
```

* js-libs terminal tab

```
nvm use 10.11.0;
cd ~/code/src/paritytech/js-libs;
git checkout luke-81-electron-4;
yarn build;
cp -r packages/electron/lib ../fether/node_modules/@parity/electron/lib;
```

* fether terminal tab

```
yarn start
```

### CODE REVIEW

**Comment by amaurymartiny** [comment]
@ltfschoen Can I close #90?

**Comment by ltfschoen** [comment]
> @ltfschoen Can I close #90?

Yes I've closed it now

**Comment by ltfschoen** [comment]
@amaurymartiny I wonder why the CI is failing with `error Couldn't find any versions for "electron" that matches "^2.0.2 | ^3.1.0 | ^4.0.1"`

**Comment by ltfschoen** [comment]
@amaurymartiny I don't think I can progress this PR without first resolving https://github.com/paritytech/js-libs/issues/58.
I've looked at the code to try and resolve https://github.com/paritytech/js-libs/issues/58 but I didn't see what could be wrong, so I was thinking the next step would be to fork js-libs, and to sign up to NPM and trying to deploy it myself to see if there are any logs that identify the issue. What do you think

File: packages/electron/package.json [hunk 1]
```diff
@@ -21,21 +21,22 @@
...
+    "electron-dl": "^1.12.0",
     "promise-any": "^0.2.0",
-    "sntp": "^3.0.1"
+    "sntp": "^3.0.2"
   },
   "devDependencies": {
     "@types/async-retry": "^1.2.1",
     "@types/checksum": "^0.1.30",
-    "electron": "^2.0.2"
+    "electron": "^2.0.2 | ^3.1.0 | ^4.0.1",
```
Comment (amaurymartiny) [inline packages/electron/package.json]: ^4.0.1 here.

Solves:
> I wonder why the CI is failing with error Couldn't find any versions for "electron" that matches "^2.0.2 | ^3.1.0 | ^4.0.1"

**Comment by ltfschoen** [comment]
@amaurymartiny It appears to be passing CI now since applying the changes to suggested, except for coveralls

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "js-libs",
   "description": "A collection of JavaScript libraries for dapp development.",
+  "version": "v3.0.26",
```
Comment (amaurymartiny) [inline package.json]: Remove this.

I just tried a `lerna version patch --push false` locally, it bumps all packages correctly, but doesn't bump this.

**Comment by amaurymartiny** [comment]
No problem for coveralls

File: packages/contracts/package.json [hunk 3]
```diff
@@ -18,12 +18,15 @@
   "scripts": {
     "docs": "typedoc && rimraf docs/README.md",
     "prebuild": "rimraf lib",
-    "build": "tsc",
+    "build": "yarn && ./node_modules/.bin/tsc",
```
Comment (amaurymartiny) [inline packages/contracts/package.json]: Sorry for the late comment, but i just (re-)saw this. With the new step 7 in the readme, `tsc` command is enough.

**Pull request merged**